provision-contest/ansible/roles/prometheus_target_web/handlers/main.yml

```diff
@@ -1,9 +1,19 @@
 ---
 - name: restart php-exporter
-  service: name=php-fpm-exporter enabled=true state=restarted
+  service:
+    name: php-fpm-exporter
+    enabled: true
+    state: restarted
+    daemon_reload: true
```

Comment:
Is this added `daemon_reload` intentional?

Reply:
Yes,

php-fpm-exporter is a service added by us, so the file is placed but `systemctl daemon-reload` would still need to be run. This option will run that step implicit which for me makes it more clear which service files are addes by us and the little overhead the 2nd time is worth it as normally we notified `daemon reload` also.